docs/assets/images/vs_code/error_message.svg

```diff
@@ -0,0 +1,3 @@
z49mx7W/75QHpy5K8ynSbDdT3OjxcKp13qnOp06zDVNY2QerQREQScIrxSqsIQAgDpEi+Qp71YF1Zyir6mtDmoTbHHa9W1jbPfwjjbwYzWJtd6hCCMJow1FDbFVQXyhgbbY//Z6svlygs6NJXsIa2+Bi0K/VZVL0hONCBALEAMQZRvrEFwgj4FsOeU2+VxZpO+ET9JjuDeQkNcKibV3DCxiuvaGqox0e+kZYD3h2oSCsYzwJUTHMMN2JcAUxx5bRc4tHWMfa9t0a1ZCWdc9xXD7MsaLN/lSh88Yqf99Xwvzm4S4yTQ37fUaZE9O8DC1bvUn6TvxQBuJ3q9wYkOxwmqaeaFP3byrctRetHLM91nuByY8pznNQu1raLcRLbOL+M1zd6xZ+vcHezZmPRVgjc9XyxWVA68tV1KViIdc+tqE9lq3eKA8pFymvTsbIYxLu5XCbQudwd4KlHQN3wdUqUsJokD1uu1guOqt0yD2XwjoStey1jcI6wPaisA4QfgKrht/zVdXLadeP00oUkXx5cytf2306hBYEq1coLa/DcwtrDNaChAcSIkdgMJgN6kEEQeUVSs2rTLMOx4ge7SjjiHrSoPf7D8qCGc56bvbjNC0BiIVjlasNEl4ibJ9Qd257pEZ88o82qAgszjXVAxmfr9PbRmjri88uqyyFx6hP2Xtk9S+qnaEuckDKiueGQYDOUtE/jj82vyBMZqQh7r2aAC8qp5Y4QXbt+U+PZGniK3vlza7rcF3DJQlfXXA/24LrUrGKdh97tGMdueHSSvq0wRUDuERKcC9BW5Q52oHwe9VxG24b8Sa8fEIglzixkPKf3yOI2IPfbzwJ12rpk4/XLl2IIBXtXOMpm3mDJUBhHSB/CusA4WfDqsMJ62x4Kjn6kMfeoyJ0HO08ChF0xd3TdUcl90nDz3TByDbOEPbbd+2Vq3vNdGfjMgmQQBgCdmdAGCk6qpjfTCQQNAEK6wBbgMI6QPjZsGoK6+zRaO4Y4hjVEO4ai77ZIIhPXkJFBal/4RnS5cYaIX604cJ4ZY+z5lGSQOIIXFr1FLml1pnaRcq4NSFcIcIWMpFA0AQorANsAQrrAOFnw6oprLNPo9mj4sVy1O8t+0mFZJwfS1bmIYGkJoDOqkM7Xx3CAK5M1/WZE9VHOmQnLpBAJhGgsM4ksLEUS2EdCyXmMQQorA2J7DG9q/GFghjTkSJxwLX5qReWyEwV5YWJBEggOoFmV1WR/q0udzJCVI9UHRy9Bm1xMnGGBBJIgMI6gbDdVVFYu4lwORKB7rfUVB0MT9RZpr37rSByBVPWJoBOUo1V3FqMtIcOaIUK5JNde/fpQSjQ4WmRinbgjjGetc+IR0cCwRK4tEpZFVXjIt2Zc/2mP2Xim1+pKCv7gj0o1k4CFgEKawtGomcprBNNnPWRAAmQAAmQAAmQQOYRoLDOPLZRS6awjoqIGUiABEiABEiABEgg2xCgsA6wqSisA4TPqkmABEiABEiABEjAZwIU1j4Djac4Cut4aDEvCZAACZAACZAACWRtAhTWAbYPhXWA8Fk1CZAACZAACZAACfhMgMLaZ6DxFEdhHQ8t5iUBEiABEiABEiCBrE2AwjrA9qGwDhA+qyYBEiABEiABEiABnwlQWPsMNJ7iKKzjocW8JEACJEACJEACJJC1CVBYB9g+FNYBwmfVJEACJEACJEACJOAzAQprn4HGUxyFdTy0mJcESIAESIAESIAEsjYBCusA24fCOkD4rJoESIAESIAESIAEfCZAYe0z0HiKK3BZT5FcueLZhXlJgARIgARIgARIgASyIoEjR2Tf4pFZ8cgkV40aNY5EO7K//vo7WpYsvT1ftVaSu1DxLH2MPDgSIAESIAESIAESIIHoBA7v2Sb7v5kZPWMAOZJCWOcpVU3ynnFVAHhZJQmQAAmQAAmQAAmQgJ8EDvy0QA5t/sbPIn0rKymENWjlrdRA8hQ/2zdwLIgESIAESIAESIAESCCxBA5tWyUHfpyX2ErjqC1phDWYwHKdp0RVyV2wGH2u47hImJUESIAESIAESIAEAiOgfKoP790uh7auyLKWasMmqYS1OWlOSYAESIAESIAESIAESMBvAhTWfhNleSRAAiRAAiRAAiRAAklJgMI6KZudJ00CJEACJEACJEACJOA3AQprv4myPBIgARIgARIgARIggaQkQGGdlM3OkyYBEiABEiABEiABEvCbAIW130RZHgmQAAmQAAmQAAmQQFISoLBOymbnSZMACZAACZAACZAACfhNgMLab6IsjwRIgARIgARIgARIICkJUFgnZbPzpEmABEiABEiABEiABPwmQGHtN1GWRwIkQAIkQAIkQAIkkJQEKKyTstl50iRAAiRAAiRAAiRAAn4ToLD2myjLIwESIAESIAESIAESSEoCFNZJ2ew8aRIgARIgARIgARIgAb8JUFj7TZTlkQAJkAAJkAAJkAAJJCUBCuukbHaeNAmQAAmQAAmQAAmQgN8EKKz9JsrySIAESIAESIAESIAEkpIAhXVSNjtPmgRIgARIgARIgARIwG8CFNZ+E2V5JEACJEACJEACJEACSUmAwjopm50nTQIkQAIkQAIkQAIk4DcBCmu/ibI8EiABEiABEiABEiCBpCRAYZ2Uzc6TJgESIAESIAESIAES8JsAhbXfRFkeCZAACZAACZAACZBAUhKgsE7KZudJkwAJkAAJkAAJkAAJ+E2AwtpvoiyPBEiABEiABEiABEggKQlQWCdls/OkSYAESIAESIAESIAE/CZAYe03UZZHAiRAAiRAAiRAAiSQlAQorJOy2XnSJEACJEACJEACJEACfhOgsPabKMsjARIgARIgARIgARJISgIU1knZ7DxpEiABEiABEiABEiABvwlQWPtNlOWRAAmQAAmQAAmQAAkkJYH/A9stHyLy8m2AAAAAAElFTkSuQmCC" preserveAspectRatio="none"/></g></g></g></g></g></svg>
```

Comment:
Nit: your SVGs are just inlined base64 armoured PNGs with nothing else. Would be better to just upload the PNGs.

Reply:
I used SVG files to maintain high resolution, but switching to PNG causes blurriness

Reply:
Interesting, lemme check what might be happening later

Reply:
Any updates on this, @axmmisaka?